### PR TITLE
Add compat data for SVG <marker>

### PR DESCRIPTION
**Pull request by JeremiePat**
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/marker

### CODE REVIEW

File: svg/elements/marker.json [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "markerUnits": {
```
Comment (SebastianZ) [inline svg/elements/marker.json]: `markerUnits` looks to me like something, which might not be supported together with the first implementation of the `<marker>` element. So, I'd rather set the support to `null` or maybe `true` after checking.

File: svg/elements/marker.json [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "viewBox": {
```
Comment (SebastianZ) [inline svg/elements/marker.json]: Same thing as for `markerUnits`. Are you sure it was supported in the initial implementation of the `<marker>` element?

File: svg/elements/marker.json [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "orient": {
```
Comment (SebastianZ) [inline svg/elements/marker.json]: Can we be sure that all browsers support this attribute?

Comment (JeremiePat) [reply svg/elements/marker.json]: It works everywhere: https://codepen.io/JeremiePat/pen/ZxpxqM

**Comment by JeremiePat** [comment]
All fixed :)

**Pull request merged**